code/modules/pda/app.dm

```diff
@@ -50,8 +50,6 @@
 		if(!pda.notifying_programs.len)
 			pda.overlays -= image('icons/obj/pda.dmi', "pda-r")
 
-/datum/data/pda/proc/
```

Comment:
whut.

Reply:
Sometime I sit and ask myself "How did this even compile"